### PR TITLE
Fixes #27275 - Fix permission bypasses in controllers

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by tbrisker** [comment]
test failures are due to rubocop

**Comment by adamruzicka** [comment]
[test foreman-tasks]

File: app/models/foreman_tasks/task/summarizer.rb [hunk 1]
```diff
@@ -47,16 +47,17 @@ def update(field, counts)
       end
     end
 
-    def initialize(recent_timeframe = 24)
+    def initialize(scope, recent_timeframe = 24)
```
Comment (iNecas) [inline app/models/foreman_tasks/task/summarizer.rb]: Possible fix for the dashboard widgets issue might making `Task.authorized` as the default for the `scope` argument

**Comment by iNecas** [comment]
All the other things seemed to work find, so waiting just for the fix for the dasboard widgets

**Comment by iNecas** [comment]
Thanks @adamruzicka

**Pull request merged**